hc/api/tests/test_admin.py

```diff
@@ -9,6 +9,9 @@ def setUp(self):
         self.check = Check.objects.create(user=self.alice, tags="foo bar")
 
         ### Set Alice to be staff and superuser and save her :)
+        self.alice.is_staff = True
+        self.alice.superuser = True
```

Comment:
This line should rather be `self.alice.is_superuser = True`. There is no `superuser` field on the User object and doing this does not grant the necessary permissions needed for user `alice` to view/edit entries in the admin. That's why 403 is being returned in the test below.